app/_src/kubernetes-ingress-controller/reference/required-permissions.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Required Permissions to Use {{ site.kic_product_name }}
```

Comment:
```suggestion
title: Required permissions to use Kong Ingress Controller
```
variables don't work right in front matter, unfortunately:
<img width="719" alt="Screenshot 2023-11-30 at 11 49 04 AM" src="https://github.com/Kong/docs.konghq.com/assets/54370747/66272246-489c-44ee-a1d4-ba1d3f1ea29d">